### DISCUSSION TITLE
Welcome to sloooop Discussions!

### DISCUSSION THREAD

**Discussion by abdulazizali77** [abdulazizali77/sloooop#66] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by BlessTheFallx** [comment]
Hello I was wondering you could update hthe Sloooop extension it used to work now it doesnt

**Reply by abdulazizali77** [reply]
Hey, thanks for the report could you post an actual bug/issue with details eg Browser, Browser version, OS etc. Havent looked at this in a long while (stopped playing in the orchestra for a while too) but ill try to find time for it
